src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -62,6 +64,23 @@ public static async Task<string> GetGitHubAppInstallationAccessToken(string gitH
             return response.Token;
         }
 
+        /// <summary>
+        /// Gets the latest release tag name of winget-create.
+        /// </summary>
+        /// <returns>Latest release tag name.</returns>
+        public static async Task<string> GetLatestRelease()

```

Comment:
You can make this non-static now that you have a githubclient in Program.cs

Reply:
Changed method to non-static